src/components/arrow/arrow.tsx

```diff
@@ -1,24 +1,17 @@
 import React from "react";
 import styled from "styled-components";
 
-interface ArrowProps {
-	ArrowComponent?: ArrowComponentType;
-	arrowStyle?: ArrowStyle;
+export interface ArrowProps {
+	style?: ArrowStyle;
 	direction: Direction;
 	onClick: () => void;
+	className?: string;
 }
 
-export type ArrowComponentType = (
-	props: ArrowComponentProps
-) => React.ReactElement;
+export type ArrowComponentType = (props: ArrowProps) => React.ReactElement;
```

Comment:
warum eigentlich nicht `React.VFC<ArrowProps>`?

Reply:
I think because we should not exlude class components at this point.
But we can shorten it to `React.ReactElement<BulletProps>`

Reply:
und überhaupt exportiert wird dieser type weil legacy?

Reply:
Export happens to give the user a type which they can use if they want to.

Reply:
Ah that makes sense, of course.
You mean `React.ElementType` I guess. I think that would actually be better as it supports null returns, refs and such.